lib/multigrid.cpp

```diff
@@ -140,13 +130,14 @@ namespace quda
     // if we aren't doing a staggered KD solve
     if (param.level != 0 || param.transfer_type == QUDA_TRANSFER_AGGREGATE) {
       // Refresh the null-space vectors if we need to
-      if (refresh && param.level < param.Nlevel - 1) {
-        if (param.mg_global.setup_maxiter_refresh[param.level]) generateNullVectors(param.B, refresh);
+      // FIXME: can we meaningfully refresh Chebyshev filter null vecs?
```

Comment:
I think the answer is yes here.  I don't think this case should be any difference from having a null space set generated from inverse iteration.  Or am I overlooking something?

Reply:
Will discuss offline